joomla/joomla-cms
# [3.5.0 Beta] Fixed storing of the permissions

## Problem

Changing permissions won't take effect and only the old settings remain.
## How to test
1. Go to System -> Global Configuration
2. Click on the Server Tab
3. Make sure Database Type is set to MySQLi
4. Click on the Permission tab
5. Change permissions and see that Joomla says they are stored
6. Reload page, permissions have old value again
7. Apply patch
8. Change permissions and see that Joomla says they are stored
9. Reload page, permissions now have the new value

This works for the global permissions and for the component permissions.
If you are using the PDO driver, there is another fix for that at #8322 

Do note, these patches cannot be used together as they change the same file.
